Dockerfile

```diff
@@ -5,6 +5,10 @@ RUN apt-get install libpq-dev -y
 RUN curl -fsSL https://get.docker.com -o get-docker.sh
 RUN sh get-docker.sh
 
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" \
+    -o /usr/local/bin/docker-compose && \
+    chmod +x /usr/local/bin/docker-compose
```

Comment:
explorer_admin uses docker-compose from machine anyway, i think we can remove it